pkg/network/msg.go

```diff
@@ -43,9 +43,9 @@ type UpdatedSchemaResponse struct {
 	ID string `json:"id"`
 }
 
-// ListThingsResponse represents the list things response
-type ListThingsResponse struct {
-	Things []*entities.Thing `json:"things"`
+// ListDevicesResponse represents the list things response
+type ListDevicesResponse struct {
+	Devices []*entities.Thing `json:"devices"`
```

Comment:
I verify the [docs](https://github.com/CESARBR/knot-fog-connector/blob/master/docs/api/amqp.md#devices-list) and notice that the expected JSON property was `devices` instead of `things`. So I changed the struct variable name either to be concised with the nomenclature. What do you thought about it??

Reply:
Personally, I prefer to use the term "thing" to talk about smart devices. We often use "devices" because that's the way everyone refers to since starting to work on the KNoT project.

In terms of code, the tags we attach to the structure exists exactly to be used when we have different representations between our internal naming and the property name we want to export in the JSON.

Reply:
I agree with @joaoaneto :smile: